snooty/test_parser.py

```diff
@@ -898,7 +898,34 @@ def test_cta_banner() -> None:
     project_config = ProjectConfig(ROOT_PATH, "", source="./")
     parser = rstparser.Parser(project_config, JSONVisitor)
 
-    # Test valid cta-banner
+    # Test valid cta-banner with specified icon
```

Comment:
My general stance is we don't need tests for just rstspec changes!

Reply:
Oh gotcha!